src/comp_loinc/__main__.py

```diff
@@ -1,5 +1,10 @@
-"""Entry point: Calls CLI"""
-from comp_loinc.build import app
-
+"""Computational LOINC"""
+try:
+    # from comp_loinc.cli import cli
+    from comp_loinc.main import app
+except ModuleNotFoundError:
+    # from .cli import cli
+    from .main import app
```

Comment:
## `__main__.py` changes
I oftentimes just use `__init__.py`, but I wasn't sure if you were importing from
Unused, un-needed file.

We can do what we need with `__init__.py`, though some add a `__main__.py` to draw some distinctions between things that are just being imported in the top level name space, vs what is supposed to happen when you run `python -m MY.PACKAGE.PATH`


Running via python -m src.comp_loinc